core/src/main/scala/org/bitcoins/core/hd/BIP32Path.scala

```diff
@@ -144,6 +149,21 @@ object BIP32Path {
 
     BIP32PathImpl(path)
   }
+
+  def fromBytes(bytes: ByteVector): BIP32Path = {
+    require(bytes.size % 4 == 0, "ByteVector is not suited for KeyPath")
+
+    val parts = bytes.toIterable.grouped(4).toVector
+
+    val path = parts.map { part =>
+      val uInt32: UInt32 = UInt32.fromBytes(ByteVector(part))
+      val hardened = uInt32 > ExtKey.hardenedIdx
```

Comment:
Should this be `>=`?

Reply:
I believe it should be: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#extended-keys 

```
The normal child keys use indices 0 through 231-1. The hardened child keys use indices 231 through 232-1. To ease notation for hardened key indices, a number iH represents i+231. 
```